runners-manager/runner/RunnerManager.py

```diff
@@ -0,0 +1,76 @@
+from runner.Runner import Runner
+from vm_creation.github_actions_api import (create_runner_token,
+                                            force_delete_runner)
+from vm_creation.openstack import create_vm, delete_vm
+from runner.VmType import VmType
+
+
+class RunnerManager(object):
+    runner_counter: int
+    github_organization: str
+    runners: dict[str, Runner]
+    runner_management: list[VmType]
+
+    def __init__(self, org: str, config: list):
+        self.runner_counter = 0
+        self.github_organization = org
+        self.runner_management = [VmType(elem) for elem in config]
+        self.runners = {}
+
+        for t in self.runner_management:
```

Comment:
Some thoughts to be noted. We shoudn't delete the runners when we reboot the service. And we don't necessarily need to create them when we boot runner manager.

We should:
* Do nothing when we reboot runner manager
* Find created VMs when we boot the runner
* Create new one if needed

Reply:
@erwan-b your task is to create a ticket for this comment :up: